lvgl/Makefile

```diff
@@ -0,0 +1,22 @@
+TOCK_USERLAND_BASE_DIR ?= ..
+LIBNAME := lvgl
+$(LIBNAME)_DIR := $(TOCK_USERLAND_BASE_DIR)/$(LIBNAME)
+
+
+DEPPATH += --dep-path $(LVGL_DIR)
+VPATH += :$(LVGL_DIR)
+CSRCS += 
+
+override CFLAGS += -Wno-error=unused-macros -Wno-error=sign-compare -Wno-error=float-equal\
+  -Wno-error=unused-parameter -Wno-error=unused-variable\
+  -Wno-error=shadow -Wno-error=type-limits -Wno-error=overflow
```

Comment:
These are disappointing, but not really our problem to fix.

Reply:
I added them when I thought the check will fail due to library warnings, I think we can remove it.